c-boyle/future-hero
# Add Pause Win Lose Menus

## Changes:
- Add a pause menu with a controls screen.
- Add a lose menu.
- Add a win menu.


## Screenshots:

Pause Menu:
![image](https://user-images.githubusercontent.com/62191940/196100685-ba6b0680-220f-48be-b04b-dcd01b3377c6.png)

Controls Panel:
![image](https://user-images.githubusercontent.com/62191940/196101336-ad913911-4af1-4fb4-8542-89e8604e5a39.png)


Loss Menu:
![image](https://user-images.githubusercontent.com/62191940/196100784-4cef2645-4fd3-4a82-a114-5cd7e561c4d2.png)

Win Menu:
![image](https://user-images.githubusercontent.com/62191940/196100953-b383536d-938a-45cc-8496-1efa0b34d4a1.png)


### How to Test:
Pause Menu:
Press Escape on keyboard to open the pause menu. Press Controls to check out the controls screen, press Escape or B on gamepad to go back to the pause menu from a submenu. Try out all the buttons.

Win Menu:
Win the level to get to the win screen. Try out it's buttons.

Lose Menu:
Lose the level to get to the lose screen. Try out it's buttons.

### Impacted Features:
Win/loss behaviour and existing win UI
